session_4/blogspot/app/views/articles/show.html.erb

```diff
@@ -0,0 +1,27 @@
+<br>
+<% unless notice.to_s.empty? %>
+  <div class="alert alert-success" role="alert">
+    <%= notice %>
+  </div>
+<% end %>
+
+<img src="<%= @article.image_url %>" alt="Invalid URL!" width=1100 height=700>
```

Comment:
Instead of specifying size in pixels, you can use `w-100` or `w-auto` bootstrap css class to expand it to the size of parent tag.

https://getbootstrap.com/docs/4.5/utilities/sizing/#relative-to-the-parent